assets/src/scripts/components/hydrograph/graph-brush.spec.js

```diff
@@ -148,7 +148,7 @@ describe ('graph-brush module', () => {
         },
         ui: {
             width: 400,
-            hydrographXRange: undefined
+            hydrographBrushOffset: undefined
```

Comment:
I think we should move this to timeSeriesState. There will be another brush soon for the dv graph.

Reply:
I moved it to time series state.